### ISSUE TITLE
Write more modules

### ISSUE THREAD

**Issue by sadboyzvone** [sadboyzvone/passthief-modules#1]
There are some modules that I've been wanting to write,but don't have the time(or I don't know if it's possible).
Take any from the list. It's okay if the module isn't cross platform,just make sure to use ```platform.system()```, to let the user know.
* [ ] Mozilla Firefox
* [ ] Internet Explorer
* [ ] Mozilla Thunderbird

Also this needs to be finished:
* [ ] Google Chrome

**Comment by sadboyzvone** [comment]
@gavinhenderson you interested? i have some modules that got both windows and linux versions but no mac

**Comment by gavinhenderson** [comment]
Yup! Looks pretty simple seen as how you've done all the hard work. Will submit a PR later today

**Comment by gavinhenderson** [comment]
@sadboyzvone Thats the PR submitted now. Also I was wondering what the point of this project? Is it just for fun or something else? I was just thinking it's not a particular vulnerability being able to see all of the passwords in plain text on a local machine as there isnt really another way of doing it unless the application was to password protect all of the remembered data.

**Comment by sadboyzvone** [comment]
It started as a fun project to mess with my friends :D
You're right it isn't supposed to exploit anything..
It's sorta a gather script pentesters can use after they compromise a system

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by gavinhenderson** [comment]
Great idea. Ill see if I can think of any more applications that store sensitive data that can be gained the same way.
